README.md

```diff
@@ -86,580 +103,13 @@ Note: The resulting "splunk-kafka-connect*.tar.gz" package is self-contained. Bu
-
-	In **props.conf**, you can set up the line breaker as follows.
-
-	```
-	[s1] # sourcetype name
-	LINE_BREAKER = (####)
-	SHOULD_LINEMERGE = false
-	```
-
-   If you don't care about the timestamp, or by default, the auto assigned timestamp is good enough, then stick to the /event HEC endpoint.
-
-4. How many tasks should I configure?
-
-	Do not create more tasks than the number of partitions. Generally speaking, creating 2 * CPU tasks per instance of Splunk Connect for Kafka is a safe estimate.
-	> Note: For example, assume there are 5 Kafka Connects running Splunk Connect for Kafka. Each host is 8 CPUs with 16 GB memory. And there are 200 partitions to collect data from. `max.tasks` will be: `max.tasks` = 2 * CPUs/host * Kafka Connect instances = 2 * 8 * 5 = 80 tasks. Alternatively, if there are only 60 partitions to consume from, then just set max.tasks to 60. Otherwise, the remaining 20 will be pending, doing nothing.
-
-5. How many Kafka Connect instances should I deploy?
-
-	This is highly dependent on how much volume per day Splunk Connect for Kafka needs to index in Splunk. In general an 8 CPU, 16 GB memory machine, can potentially achieve 50 - 60 MB/s throughput from Kafka into Splunk if Splunk is sized correctly.
-
-6. How can I track data loss and data collection latency?
-
-	Please refer to the **Data loss and latency monitoring** section.
-
-7. Is there a recommended deployment architecture?
-
-	There are two typical architectures.
-
-	* Setting up a heavy forwarder layer in front of a Splunk platform indexer cluster in order to offload the data injection load to your Splunk platform indexer cluster. Setting up a heavy forwarder layer has performance benefits for the Splunk search app.
-
-		Kafka Connect Cluster (in containers or virtual machines or physical machines) -> Heavy Forwarders (HEC) -> Splunk Indexer Cluster
-
-	* Direct inject data to Splunk Indexer cluster
-
-		Kafka Connect Cluster (in containers or virtual machines or physical machines) -> Splunk Indexer Cluster (HEC)
-
-
-## Troubleshooting
-
-1. Append the **log4j.logger.com.splunk=DEBUG** to **config/connect-log4j.properties** file to enable more verbose logging for Splunk Connect for Kafka.
-2. Kafka connect encounters an "out of memory" error. Remember to export environment variable **KAFKA\_HEAP\_OPTS="-Xmx6G -Xms2G"**. Refer to the [Deployment](#deployment) section for more information.
-3. Can't see any Connector information on third party UI. For example, Splunk Connect for Kafka is not shown on the Confluent Control Center. Make sure cross origin access is enabled for Kafka Connect. Append the following two lines to connect configuration, e.g. `connect-distributed.properties` or `connect-distributed-quickstart.properties` and then restart Kafka Connect.
-
-	```
-	access.control.allow.origin=*
-	access.control.allow.methods=GET,OPTIONS,HEAD,POST,PUT,DELETE
-	```
+* Deployment
```

Comment:
Keep the format of these being sections within the README.
Small one liner with a hyperlink to each section. 
We can't just do bullet points make these look like sections

Reply:
Done